src/main/webapp/nearmeScript.js

```diff
@@ -83,7 +86,9 @@ function isNearby(geocoder, event, locationCircle, userId) {
       if (userId) {
         if (event.rsvpAttendees.includes(userId) ||
         event.invitedAttendees.includes(userId) || 
-        event.privacy == 'public') { 
+        event.privacy == 'public') {
+          // get the rating of the event location
+          // create object with event and rating and add to working list of objects. The list will be a global variable
```

Comment:
if it would be easier for you, you could try to pass the location rating to the create event form when you pass the location name and ID? Then I could store the rating with the events, and you could then just sort all the events by the rating then?